### DISCUSSION TITLE
Question: What is "Needle" referring to?

### DISCUSSION THREAD

**Discussion by librarianmage** [fabrizioschiavi/pragmatapro#254] [Q&A] [answered]
Hello,

On the font page there is a section in the Programming Language / Context section titled "Needle". What is that referring to? A programming language, a program, a type of diagram? My searches are failing me.

![](https://fsd.it/wp-content/uploads/2015/12/PragmataPro_Needle.png.webp)

**Comment by fabrizioschiavi** [comment]
I've no big knowledge on programming languages but I think [this](https://www.nongnu.org/needle/) is the official Needle page

**Reply by librarianmage** [reply]
That language looks quite different from the photo, though 🤔

**Reply by fabrizioschiavi** [reply]
Are you already tried it?

**Reply by librarianmage** [reply]
https://cvs.savannah.nongnu.org/viewvc/needle/needle/ none of this looks like the image

**Reply by fabrizioschiavi** [reply]
If I remember correctly, I found this screenshot from Reddit years ago. 
I looking for it in these days but I haven't been lucky. Strange…
At this point I'm very curious about this Needle mistery. Please let me know if you'll find something. Thanks!

**Comment by librarianmage** [comment, marked as answer]
Found it! http://scrambledeggsontoast.github.io/2014/09/28/needle-announce/

Edit: added link because I submitted too quickly

**Reply by fabrizioschiavi** [reply]
You've right! Thanks for this sharing!
